### ISSUE TITLE
Sorry, you have been blocked. You are unable to access pinata.cloud

### ISSUE THREAD

**Issue by codetyper** [anticensority/runet-censorship-bypass#335]
Не работает PAC-файл, потому что pinata.cloud блокирует запросы из РФ.

**Comment by ilyaigpetrov** [comment]
Да, всё верно, на эту тему была опубликована [новость](https://github.com/anticensority/runet-censorship-bypass/issues/10#issuecomment-2524588989).

**Comment by sk8rthug** [comment]
А если расширение вместе с PAC-скриптом уже давно установлены, почему расширение всё равно не работает?

**Comment by sk8rthug** [comment]
А если расширение вместе с PAC-скриптом уже давно установлены, почему расширение всё равно не работает? Или этот скрипт не хранится на компе, и нужно, чтоб он каждый раз при запуске по-новой подтягивался? Если так, то в чём проблема хранить его где-нибудь в другом месте? Зачем нам эта пината? Если он и так  давно не обновляется, почему нельзя хранить его локально?

**Comment by SyntaxEvg** [comment]
как установить вручную эти pac скрипты из pinata.cloud ?и где их брать, почему вы не разметили эти скрипты на сайте github ?

**Comment by Verity-Freedom** [comment]
> как установить вручную эти pac скрипты из pinata.cloud ?и где их брать, почему вы не разметили эти скрипты на сайте github ?

В [Tor Portable](https://github.com/Verity-Freedom/Tor-Portable) Антизапрет работает, даже если сломаны адреса обновлений.
